lib/particle.js

```diff
@@ -0,0 +1,20 @@
+const { Vec3 } = require("vec3")
+
+module.exports = loader
+
+function loader (registry) {
+  function fromNetwork(packet){
```

Comment:
I didn't mean to get rid of the class. I meant to make fromNetwork a static method inside the class.

Reply:
Apologies, reverted to class with static function